website/docs/docs/deploy/cloud-ci-job.md

```diff
@@ -51,19 +61,21 @@ The green checkmark means the dbt builds and tests were successful. Clicking the
 The green checkmark means the dbt builds and tests were successful. Clicking on the dbt Cloud section navigates you to the relevant CI run in dbt Cloud.
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/Enabling-CI/ADO CI Check.png" title="Azure DevOps pull request"/>
 
-## Configuring a dbt Cloud CI job
+## Configuring a Slim CI job
 
-Setting up a CI job is very similiar to setting up a normal production job that runs on a schedule; however, a CI job has some notable differences.
+Slim CI offers an alternative to running and testing all models in your project, which costs time (and money) and feels especially painful if your pull request only proposes changes to a handful of models. Slim CI enables you to run modified models so you can focus on testing those changes. 
```

Comment:
```suggestion
Slim CI offers an alternative to dbt Cloud for running and testing all models in your project, which costs time (and money) and feels especially painful if your pull request only proposes changes to a handful of models. Slim CI enables you to run modified models so you can focus on testing those changes. 
```

Reply:
sugg: consider rewording so it's not so _opinionated_ about Slim CI (eg maybe flip it around and talk about our benefits -- opinionated about our tech stack). i see that more in blogs, less so in product docs.